chart/templates/NOTES.txt

```diff
@@ -1 +1,17 @@
 1. The kapel accounting cron job should be running now. When the kapel pod runs check the logs of the processor and ssmsend containers.
+
+{{ if not .Values.processor.config.NAMESPACE -}}
+WARNING: processor.config.NAMESPACE is empty in values.yaml. Set it to the namespace in the cluster where your workload pods run.
+{{ end -}}
+
+{{ if not .Values.processor.config.SITE_NAME -}}
+WARNING: processor.config.SITE_NAME is empty in values.yaml. Set it to the name of the site this cluster is in.
+{{ end -}}
+
+{{ if not .Values.processor.config.SUBMIT_HOST -}}
+WARNING: processor.config.SUBMIT_HOST is empty in values.yaml. Set it to a uniquely identifying domain name for this cluster.
+{{ end -}}
+
+{{ if not .Values.processor.config.VO_NAME -}}
+WARNING: processor.config.VO_NAME is empty in values.yaml. Set it to the Virtual Organization the site is participating in.
```

Comment:
VO_NAME still has an uncommented default value "example-vo"  which I suppose is fine, anything published for "example-vo" should be considered invalid or ignored. But that being the case this warning is not needed because it will never be empty.